scripts/src/java/org/oppia/android/scripts/common/GitClient.kt

```diff
@@ -36,7 +36,11 @@ class GitClient(
   }
 
   private fun retrieveBranchMergeBase(): String {
-    return executeGitCommandWithOneLineOutput("merge-base $baseDevelopBranchReference HEAD")
+    return executeGitCommandWithOneLineOutput("merge-base $baseCommit HEAD").also {
+      if (baseCommit != it) {
+        println("WARNING: Provided base commit $baseCommit doesn't match merge-base: $it.")
```

Comment:
TODO: explain in PR description why this change was added (peace of mind due to functional differences from previous version that checks against the develop branch).

Reply:
Done.